templates/cli/src/Parser.php.twig

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace {{ spec.title | caseUcfirst }};
+
+use jc21\CliTable;
+use jc21\CliTableManipulator;
+
+class Manipulators extends CliTableManipulator {
+    public function truncate($value) {
+		if (is_bool($value)) {
+            $value = $value ? 'true' : 'false';
+        }
+		return strlen(empty($value) ?? '') > 15 ? substr($value,0,15).'...' : $value;
+    }    
+}
+
+class Parser {
+
+    private const colors = array(
+        'blue',
+        'red',
+        'green',
+        'yellow',
+        'magenta',
+        'cyan',
+        'white',
+        'grey'
+    );
+
+    private const tableColor = self::colors[0];
+    private const headerColor = self::colors[2];
+
+    public function parseResponse($response, $properties = array()) {
```

Comment:
Would be nice to add doc blocks to this classes.

Reply:
Done.